build/parent/pom.xml

```diff
@@ -42,14 +42,14 @@
         <ivyde-site>https://dist.apache.org/repos/dist/release/ant/ivyde/updatesite/ivyde-2.2.0.final-201311091524-RELEASE/</ivyde-site>
         <liferay-ide-swtbot-testing-site>https://files.liferay.org.es/staged/public-files/liferay-ide/unstable/build/com.liferay.ide.testing-repository/target/repository/</liferay-ide-swtbot-testing-site>
         <liferay-ide-tag>ga1</liferay-ide-tag>
-        <m2e-site>https://download.eclipse.org/technology/m2e/releases/1.13/1.13.0.20190716-1624/</m2e-site>
+        <m2e-site>https://archive.eclipse.org/technology/m2e/releases/1.13/1.13.0.20190716-1624/</m2e-site>
```

Comment:
The team should probably update to a later m2e release. I think they are already up to 1.18 or 1.19 now.

Reply:
The m2e 1.18 and 1.19 can't be installed successfully since we're using the 201906 as the base eclipse. Eclipse 202012 requiers jdk11 and from 202012 it bundles a jre by default. Should we update base eclipse to 202009 and then update m2e as well?